spec/models/search/bookmark_search_form_spec.rb

```diff
@@ -13,6 +13,45 @@
 
   describe "bookmarkable_search_results" do
     describe "sorting" do
+      context "by word count" do
+        let(:tag) { create(:canonical_fandom) }
+
+        let!(:work5) { create(:work, fandom_string: tag.name, chapter_attributes: { content: "one two three four five" }) }
+        let!(:work10r) { create(:work, fandom_string: tag.name, restricted: true, chapter_attributes: { content: "one two three four five six seven eight nine ten" }) }
+        let!(:work10) { create(:work, fandom_string: tag.name, title: "Ten", chapter_attributes: { content: "one two three four five six seven eight nine ten" }) }
+        # work "Ten" has word_count 10
+        let!(:work_bookmark) { create(:bookmark, bookmarkable: work10) }
+
+        let!(:series) { create(:series, title: "Series to be bookmarked") }
+        let!(:serial_work1) { create(:serial_work, series: series, work: work5) }
+        let!(:serial_work2) { create(:serial_work, series: series, work: work10r) }
+        # series "Series to be bookmarked" word_count is 5 or 15
+        let!(:series_bookmark) { create(:bookmark, bookmarkable: series) }
```

Comment:
Is this the recommended way to set up the series/works I want for a test? I'm not sure whether the reason the associated test fails is due to a bug in the code, a problem with the test, or a problem with the test setup :thinking: